backend/app/views/spree/admin/customer_returns/_return_item_decision.html.erb

```diff
@@ -39,7 +39,9 @@
           <%= return_item.acceptance_status_errors %>
         </td>
         <td class-"align-center">
-          <%= return_item.reception_status.humanize %>
+          <span class="pill pill-<%= return_item.reception_status %>">
+            <%= return_item.reception_status.humanize %>
```

Comment:
We should only use `humanize` if we pass it through `I18n` before and humanize the fallback translation.

```ruby
Spree.t(return_item.reception_status, 
  scope: 'return_item_reception_statuses', 
  default: return_item.reception_status.humanize)
```